vcfa/resource_vcfa_vcenter.go

```diff
@@ -15,6 +15,8 @@ import (
 
 const labelVcfaVirtualCenter = "Tenant Manager vCenter Server"
 
+const extraSleepAfterOperations = 3 * time.Second
```

Comment:
This is https://github.com/vmware/terraform-provider-vcfa/pull/9#issuecomment-2609045641

Reply:
Good we have TODO: TM: there below!